Common/UnitDefinitions/VolumePerLength.json

```diff
@@ -100,6 +100,30 @@
           "Abbreviations": [ "yd³/ftUS" ]
         }
       ]
+    },
+    {
+      "SingularName": "UsGallonPerMile",
+      "PluralName": "UsGallonsPerMile",
+      "FromUnitToBaseFunc": "{x} / (1000 * 1609.344 / 3.785411784)",
+      "FromBaseToUnitFunc": "{x} * (1000 * 1609.344 / 3.785411784)",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "gal (U.S.)/mi" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "ImperialGallonPerMile",
```

Comment:
@tmilnthorp I'm seeing both Imperial and UK, almost 50/50.
Should we decide on a convention and start cleaning house?

I like "UK", since it complements "US". Wolfram also uses US gallon and UK gallon.

https://www.wolframalpha.com/input?i=1.0000000000000000+cubic+meter+per+meter+to+imperial+gallon+per+mile

![image](https://user-images.githubusercontent.com/787816/229283697-ac6d5772-070c-4b42-97ee-620c2f1dc570.png)

Reply:
Is there an issue to track breaking tech debts for the next major release?
It is a minor thing, but some units has strange name with prefixes, for example: MegausGallons
I suggest to use uppercase character after prefixes for all units like: MegaUsGallons, MicroLiters etc.

Reply:
v6 Wishlist #1200

The naming convention is PascalCase for words.
I agree MegausGallons reads a bit wonky, but Microliter is a single word and correct by convention.

We could probably add a special-case for a few names, like forcing uppercase for `Us` and `Uk`, but generally the convention works quite well I think.

Reply:
Let's merge this as it is. If we decide on a convention for imperial/UK and US units, it is a breaking change any way that would have to wait for #1200. We'll address all those units one go then.

Reply:
Added to wishlist: https://github.com/angularsen/UnitsNet/issues/1200#issuecomment-1493079083